models/migrations/v1_19/v233.go

```diff
@@ -56,9 +56,9 @@ func batchProcess[T any](x *xorm.Engine, buf []T, query func(limit, start int) *
 func AddHeaderAuthorizationEncryptedColWebhook(x *xorm.Engine) error {
 	// Add the column to the table
 	type Webhook struct {
-		ID   int64            `xorm:"pk autoincr"`
-		Type webhook.HookType `xorm:"VARCHAR(16) 'type'"`
-		Meta string           `xorm:"TEXT"` // store hook-specific attributes
+		ID   int64                   `xorm:"pk autoincr"`
+		Type webhook_module.HookType `xorm:"VARCHAR(16) 'type'"`
```

Comment:
```suggestion
		Type string `xorm:"VARCHAR(16) 'type'"`
```

why not resolve type to make migrations independent...

Reply:
Hmm… What's the opinion of others on this?
I'm neutral, I can see advantages for both.

Reply:
@lunny @KN4CK3R preferences?

Reply:
Most/All(?) migrations use the resolved type instead of the real type to prevent errors if the referenced type changes.

Reply:
Done